<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMSettings</class>
 <widget class="QMainWindow" name="IHMSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>447</width>
    <height>324</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>447</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Circuit d'eau - Paramètres</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Cuve Rigide</string>
    </property>
    <widget class="QSpinBox" name="diametre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>1500</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelDiametre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Taille Diamètre</string>
     </property>
    </widget>
    <widget class="QLabel" name="uniteDiametre">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="longueur">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelLongueur">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Taille Longueur</string>
     </property>
    </widget>
    <widget class="QLabel" name="uniteLongueur">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="estimationLitre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume estimée: 3534.29 L</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Remplissage_max">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>51</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt; % &lt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="Remplissage_min">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>49</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Taux de
remplissage:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Cuve Souple</string>
    </property>
    <widget class="QDoubleSpinBox" name="VolumeSouple">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="maximum">
      <double>1000000000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelVolume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume:</string>
     </property>
    </widget>
    <widget class="QLabel" name="estimationLitreSouple">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume estimée: 10000 L</string>
     </property>
    </widget>
    <widget class="QLabel" name="uniteVolume">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>m3</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveAndQuit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder et quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
